### DISCUSSION TITLE
Have a way to handle a event from window title bar?

### DISCUSSION THREAD

**Discussion by allan-rcos** [ocornut/imgui#7366] [New Users Build/Link/Run issues ONLY!]
I'm new in C++ and ImGui, so I'm not finded a exemple to it:

I need to handle a double click event in the window titlebar. My idea was create a way to the user change the title with double click, so the window title would to be a input. But I know that it can be hard to implement, so if I show a modal asking the user a new window name, solve my problem. But I don't know how to add this event for the window title...

Thank you everyone!

**Comment by ocornut** [comment]
Please open an issue to ask question, as written in the title of this section.

**Reply by allan-rcos** [reply]
Ok

**Reply by allan-rcos** [reply]
https://github.com/ocornut/imgui/issues/7369
